lighthouse-core/test/scoring-test.js

```diff
@@ -35,7 +35,7 @@ describe('ReportScoring', () => {
   describe('#scoreAllCategories', () => {
     it('should score the categories', () => {
       const resultsByAuditId = {
-        'my-audit': {rawValue: 'you passed', score: 0},
+        'my-audit': {score: 0},
         'my-boolean-audit': {score: 1, extendedInfo: {}},
```

Comment:
man, never thought I'd see the day where extendedInfo outlives rawValue 😆

Reply:
> man, never thought I'd see the day where extendedInfo outlives rawValue 😆

hahaha, maybe it's time to do the final purge on that